ui/src/components/Document/DocumentTitle.js

```diff
@@ -1,19 +1,54 @@
 import { DocumentEdition } from '@components/Document';
 import React from 'react';
+import PropTypes from 'prop-types';
 import Truncate from 'react-truncate';
+import _get from 'lodash/get';
 
-export const DocumentTitle = ({ document, short, truncate }) => {
-  const documentMetadata = document.metadata ? document.metadata : document;
-  const title = short ? (
-    documentMetadata.title
-  ) : (
-    <>
-      {documentMetadata.title} (<DocumentEdition document={document} />{' '}
-      {documentMetadata.publication_year})
-    </>
-  );
+const EditionTitleCmp = ({ metadata }) => {
+  const edition = _get(metadata, 'edition');
+  const year = _get(metadata, 'publication_year');
+  /* render both edition and year, or only edition, or only year or nothing
+   * title (edition - year)
+   * title (edition)
+   * title (year)
+   * title
+   */
+  const editionYearCmp =
+    edition && year ? (
+      <>
+        {'('}
+        <DocumentEdition metadata={metadata} /> - {year}
+        {')'}
```

Comment:
Is this valid `(<DocumentEdition metadata={metadata} /> - {year})`?

Reply:
not in JSX unfortunately, the `(` is user for the language, otherwise I would have fixed that ugliness!